### PR TITLE
Feature/section9: Socioeconomic Governance and Status Outcomes

### PR DESCRIPTION
**Pull request by julia-rom**
- All sections excluding tabular input (9.4). Will create a separate pr for this section
-  Bug fix for section 8 where page was crashing due to typo in questions file

### CODE REVIEW

File: mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    changeInGovernance: yup.string(),
+    currentGovenance: multiselectWithOtherValidationNoMinimum,
+    changeInTenureArrangement: yup.string(),
+    currentLandOwnership: multiselectWithOtherValidationNoMinimum,
+    rightsToLandInLaw: yup.string(),
+    achievementOfSocioeconomicAims: yup.string()
+  })
+  const reactHookFormInstance = useForm({
+    defaultValues: {
+      currentGovenance: { selectedValues: [], otherValue: undefined },
```
Comment (mmnoo) [inline mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js:41]: is `otherValue` needed? Doesnt look like this question needs it.

File: mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            />
+            <ErrorText>{errors.currentGovenance?.selectedValues?.message}</ErrorText>
+          </FormQuestionDiv>
+        ) : null}
+        <FormQuestionDiv>
+          <StickyFormLabel>{questions.changeInTenureArrangement.question}</StickyFormLabel>
+          <Controller
+            name='changeInTenureArrangement'
+            control={control}
+            defaultValue=''
```
Comment (mmnoo) [inline mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js:165]: Nit: If this still works when you dont have the inline defaultValue (as in not a bug fix to MUI), I think its better to be consistent with where we declare defaults and have them in the defaultValues object.

**Review by mmnoo — APPROVED** [review]

File: mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js (hunk 1 above)
Comment (julia-rom) [reply mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js:41]: yeah good call, will remove `otherValue`

File: mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js (hunk 2 above)
Comment (julia-rom) [reply mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js:165]: agreed that its good to be consistent. The controller often bugs out without a defaultValue though

**Pull request merged**